en_US.json

```diff
@@ -1488,6 +1488,7 @@
 	"property.edgeLogic.shortCut": "Keyboard Shortcuts: ",
 	"property.edgeLogic.showHelp": "show document about the directive at the cursor",
 	"property.edgeLogic.softWrap": "Soft Wrap Text",
+	"property.enableIPv6Origin.alertMessage.usedIpv6OriginServer": "You've used an IPv6 address in the origin server. Your property must enable access to origin using IPv6.",
```

Comment:
Suggested text change:

Since you have specified an IPv6 address for an origin server,  you must set "Enable IPv6 Origin" to "enable".

Reply:
We have used "You've" in other alert messages e.g. for shields. So retained the same style.

I will update the shield related alert message to use "You have" instead of "You've".

Reply:
Updated as per suggestion.